helper/strutil/strutil.go

```diff
@@ -174,6 +174,16 @@ func ParseArbitraryStringSlice(input string, sep string) []string {
 	return ret
 }
 
+// TrimStrings takes a slice of strings and returns a slice of strings
+// with trimmed spaces
+func TrimStrings(items []string) []string {
```

Comment:
I *think* that since slices are reference types, you don't even need to allocate at all! You can just update in-place, not unlike how `sort.Strings` does it.